examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -859,7 +859,7 @@
               "mfgCode": null,
               "side": "server",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "External",
```

Comment:
Should make this change to all .zap files that include Descriptor, no?  And to all endpoints that have Descriptor on them.

Reply:
I updated all .zap file.